include/openssl/kdf.h

```diff
@@ -28,7 +28,7 @@ extern "C" {
 # define EVP_PKEY_CTRL_SCRYPT_N                 (EVP_PKEY_ALG_CTRL + 10)
 # define EVP_PKEY_CTRL_SCRYPT_R                 (EVP_PKEY_ALG_CTRL + 11)
 # define EVP_PKEY_CTRL_SCRYPT_P                 (EVP_PKEY_ALG_CTRL + 12)
-# define EVP_PKEY_CTRL_SCRYPT_MAXMEM_MIB        (EVP_PKEY_ALG_CTRL + 13)
+# define EVP_PKEY_CTRL_SCRYPT_MAXMEM_BYTES      (EVP_PKEY_ALG_CTRL + 13)
```

Comment:
I'd suggest macros for these too. So something like EVP_PKEY_CTX_set_scrypt_N etc. as we do with the TLS PRF. The uint64_t cases maybe a new wrapper function that takes a uint64_t parameter and passes it to the underlying ctrl as a pointer?